src/main/java/org/spongepowered/common/data/fixer/PlayerRespawnData.java

```diff
@@ -0,0 +1,75 @@
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.data.fixer;
+
+import com.flowpowered.math.vector.Vector3d;
+import com.google.common.collect.Maps;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraft.util.datafix.IFixableData;
+import org.spongepowered.api.util.RespawnLocation;
+import org.spongepowered.common.data.util.DataVersions;
+import org.spongepowered.common.data.util.NbtDataUtil;
+
+import java.util.Map;
+import java.util.UUID;
+
+public class PlayerRespawnData implements IFixableData {
+
+    @Override
+    public int getFixVersion() {
+        return DataVersions.PlayerData.RESPAWN_DATA_1_9_VERSION;
+    }
+
+    @Override
+    public NBTTagCompound fixTagCompound(NBTTagCompound compound) {
+        final Map<UUID, RespawnLocation> spawnLocations = Maps.newHashMap();
+
+        if (compound.hasKey(NbtDataUtil.USER_SPAWN_X, NbtDataUtil.TAG_ANY_NUMERIC)
+            && compound.hasKey(NbtDataUtil.USER_SPAWN_Y, NbtDataUtil.TAG_ANY_NUMERIC)
+            && compound.hasKey(NbtDataUtil.USER_SPAWN_Z, NbtDataUtil.TAG_ANY_NUMERIC)) {
+            Vector3d pos = new Vector3d(compound.getInteger(NbtDataUtil.USER_SPAWN_X),
+                    compound.getInteger(NbtDataUtil.USER_SPAWN_Y),
+                    compound.getInteger(NbtDataUtil.USER_SPAWN_Z));
+//            final UUID key = WorldPropertyRegistryModule.dimIdToUuid(0);
+//            spawnLocations.put(key, RespawnLocation.builder().world(key).position(pos).build());
+            // This is the point where we need to check the old data, if it is available.
+        }
+        NBTTagList spawnlist = compound.getTagList(NbtDataUtil.USER_SPAWN_LIST, NbtDataUtil.TAG_COMPOUND);
+        // This is legacy forge versions, not sure how forge is going to be saving it from now on, but
```

Comment:
What needs to be changed?
Forge is still storing it the same way
https://github.com/MinecraftForge/MinecraftForge/blob/1.9/patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch#L152-L184